data/sdo-howto-examples.txt

```diff
@@ -94,94 +94,95 @@ MICRODATA:
-                        <div itemprop="text">Remove the flat tire and put the spare tire on the exposed lug bolts.</div>
-                    </div>
-                    <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToDirection">
-                        <meta itemprop="position" content="6"></meta>
-                        <div itemprop="text">Tighten the lug nuts by hand.</div>
-                    </div>
-                    <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToTip">
-                        <meta itemprop="position" content="7"></meta>
-                        <div itemprop="text">Don't use the wrench just yet.</div>
-                    </div>
-                </div>
-            </div>
-            <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToSection">
-                <div itemprop="name">Finishing up</div>
-                <meta itemprop="position" content="3"></meta>
-                <div itemprop="steps" itemscope itemtype="http://schema.org/HowToStep">
-                    <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToDirection">
-                        <meta itemprop="position" content="1"></meta>
-                        <div itemprop="text">Lower the jack and tighten the lug nuts with the wrench.</div>
-                    </div>
-                    <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToDirection">
-                        <meta itemprop="position" content="2"></meta>
-                        <div itemprop="text">Replace the hubcap.</div>
-                    </div>
-                    <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToDirection">
-                        <meta itemprop="position" content="3"></meta>
-                        <div itemprop="text">Put the equipment and the flat tire away.</div>
-                    </div>
-                </div>
-            </div>
+    <div itemprop="tool">Spare tire</div>
+    <div itemprop="tool" itemscope itemtype="http://schema.org/HowToTool">
+        <span itemprop="name">Lug wrench</span>
+        <img itemprop="image" src="lug-wrench.jpg"/>
+    </div>
+    <div itemprop="tool">Jack</div>
+    <div itemprop="tool" itemscope itemtype="http://schema.org/HowToTool">
+        <span itemprop="name">Wheel wedges</span>
+        <img itemprop="image" src="wheel-wedges.jpg"/>
+    </div>
+    <div itemprop="supply" itemscope itemtype="http://schema.org/HowToSupply">
+        <span itemprop="name">Flares</span>
+        <img itemprop="image" src="flares.jpg"/>
+    </div>
+    <div itemprop="step" itemscope itemtype="http://schema.org/HowToSection">
+        <div itemprop="name">Preparation</div>
+        <meta itemprop="position" content="1"/>
+        <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToDirection">
```

Comment:
I believe there should be a HowToStep wrapping this HowToDirection + HowToTip.
So the structure would be:
HowTo
  - HowToSection
     - HowToStep (aka "step 1")
        - HowToDirection
        - HowToTip
     - HowToStep (aka "step 2")
        - HowToDirection
etc

Maybe a bit verbose, but the Direction + Tip I believe were intended to be parts of an individual Step.